### DISCUSSION TITLE
Actividad 0

### DISCUSSION THREAD

**Discussion by ruipablo** [frlp-utn-ingsoft/catedra#60] [Show and tell]
https://github.com/ruipablo/Actividad0.git

**Comment by RodrigoJacznik** [comment]
Hola,
no pude probar la app. Encontré los siguientes errores:

1. en `mysite/urls.py` usas `pools` en lugar de `polls`
2. los 3 templates están mal. La idea del tutorial es ir evolucionando el código, y el código viejo no debe quedar. Por ejemplo: en `templates/index.html` la linea 10 no iría
3. en `views.py` te quedaron funciones repetidas. Por ejemplo hay 2 funciones y 1 clase para `index`. Al igual que en el punto anterior, tenes que ir reemplazando el código viejo.

Arregla estos puntos para aprobar la actividad.

**Comment by ruipablo** [comment]
Buenas Rodrigo, ya realicé los cambios.

**Reply by RodrigoJacznik** [reply]
Bien, ahora funciona pero la db que pusheaste no tiene ninguna poll. Le agregas alguna para poder testear la votación? Eso se hace en esta parte del tutorial: https://docs.djangoproject.com/en/5.0/intro/tutorial02/#playing-with-the-api

**Comment by ruipablo** [comment]
Buen día Rodrigo. Cambié lo que me decías, probe la app y funciona bien.

**Reply by RodrigoJacznik** [reply]
Pudiste votar en alguna encuesta?
Cuando corro la app me aparece el mensaje "No polls are available."

**Comment by ruipablo** [comment]
si yo pude votar hoy, te paso screenshot
![127 0 0 1_8000_polls_1_ - Google Chrome 16_4_2024 12_57_45](https://github.com/frlp-utn-ingsoft/catedra/assets/166929193/1141e570-b5ad-47ca-ad0a-f9ce020f0be5)
![127 0 0 1_8000_polls_1_ - Google Chrome 16_4_2024 12_57_53](https://github.com/frlp-utn-ingsoft/catedra/assets/166929193/146172c8-519a-4d36-aa34-ca5a60ce2bef)
![127 0 0 1_8000_polls_1_ - Google Chrome 16_4_2024 12_58_02](https://github.com/frlp-utn-ingsoft/catedra/assets/166929193/e510cac2-159a-4af9-9799-8ef365a3ac48)
![WhatsApp Image 2024-04-16 at 1 04 08 PM](https://github.com/frlp-utn-ingsoft/catedra/assets/166929193/78e9eb12-cb43-41e1-ac37-3b5e9e21b1dd)

**Reply by RodrigoJacznik** [reply]
Fijate si no te quedo sin pushear la DB porque la que esta en el repo no tiene tablas

**Comment by ruipablo** [comment]
ahh si si perdon me falto eso, Ya esta!

**Reply by RodrigoJacznik** [reply]
Ahora si.
La actividad esta aprobada.
